client/src/routes/Menu/index.jsx

```diff
@@ -32,11 +32,18 @@ const {
   emptySmallEllipse,
   fullSmallEllipse,
   marginLeft15px,
+  img,
+  imgWrapper,
+  companyName,
+  companyWrapper,
 } = styles;
 
 function Menu() {
   return (
     <Container>
+      <div className={imgWrapper}>
+        <img src="./assets/logo.svg" alt="guy in vr" className={img} />
+      </div>
```

Comment:
In lines 53, 58, 63, 68, and 73, backticks are used for the className but only using one variable. It could just be removed.

Reply:
Changes were made for lines 53, 58, 63, 68, and 73. 

Alt tag content added for 79 and 80.